participants/luke-nailer/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Luke Nailer</title>
+  <link rel="stylesheet" href="index.css">
```

Comment:
make sure the css file name is correct. this should be:
`<link rel="stylesheet" href="style.css">`